server/sonar-server/src/test/java/org/sonar/server/notification/ws/AddActionTest.java

```diff
@@ -122,6 +123,7 @@ public void add_to_a_specific_channel() {
   @Test
   public void add_a_project_notification() {
```

Comment:
this method should be named "add_notification_on_private_with_USER_permission" and a test called "add_notification_on_public_project" should be added

public project do not require USER permission to be added explicitly but I don't this being tested anywhere